lib/rucio/common/utils.py

```diff
@@ -503,7 +503,7 @@ def generate_key(*arg, **kw):
     return generate_key
 
 
-def construct_surl_DQ2(dsn, filename):
+def construct_surl_DQ2(dsn, scope, filename):
```

Comment:
I will create a PR to remove this function since it is not used anymore #5587 . If you want to add this before your vacation, just change it. :+1:

Reply:
This is actually still used, so don't remove 😄

Reply:
Whoops okay. I did not forget to include it in #5587, it was completely on purpose then! :sweat_smile:

Reply:
I knew before the merge of that PR that I'll quickly become crazily annoyed by these type annotations ...

Reply:
Yes, we all will be! haha
Sorry not sorry :raising_hand_man: